code/controllers/subsystem.dm

```diff
@@ -168,7 +168,7 @@
 		queue_node_priority = queue_node.queued_priority
 		queue_node_flags = queue_node.flags
 
-		if (queue_node_flags & SS_TICKER)
+		if (queue_node_flags & (SS_TICKER|SS_BACKGROUND)) == SS_TICKER)
 			if ((SS_flags & (SS_TICKER|SS_BACKGROUND)) != SS_TICKER)
 				continue
```

Comment:
This bit's redundant now

Reply:
no its not, re-read it

Reply:
Ah I'm dumb. That's always heartwarming to discover